scripts/validate_docstrings.py

```diff
@@ -470,21 +502,29 @@ def main(func_name, prefix, errors, output_format, ignore_deprecated, ignore_fun
         "all docstrings",
     )
     argparser.add_argument(
-        "--ignore_functions",
-        nargs="*",
-        help="function or method to not validate "
-        "(e.g. pandas.DataFrame.head). "
-        "Inverse of the `function` argument.",
+        "--for_error_ignore_functions",
```

Comment:
I'd personally use `--ignore_errors` for the argument and just `ignore_errors` or `errors_to_ignore` the variable name. To me `for_error_ignore_functions` is confusing.

Reply:
Done. Thanks for the good suggestion.